cluster/gce/configure-vm.sh

```diff
@@ -254,6 +254,13 @@ function create-salt-auth() {
   kubelet_auth_file="/srv/salt-overlay/salt/kubelet/kubernetes_auth"
   (umask 077;
     echo "{\"BearerToken\": \"${KUBELET_TOKEN}\", \"Insecure\": true }" > "${kubelet_auth_file}")
+
+  # Generate tokens for other "service accounts".  Append to known_tokens.
+  for a in "system:scheduler" "system:controller_manager" "system:logging" "system:monitoring" "system:dns"
```

Comment:
I would make this a constant array, and lift the "do" up. Just random style nit.
```
local -r service_accounts=("system:scheduler" "system:controller_manager" "system:logging" "system:monitoring" "system:dns")
for account in "${service_accounts[@]}"; do

done
```